README.md

```diff
@@ -5,7 +5,9 @@
 
 > [!NOTE]
 > Documentation is all on our website [here](https://part-of.my.id/docs#structure).
-
+<p align="center">
 <a href="https://discord.gg/rFyRF3MMhc"><img alt="Discord Server" src="https://invidget.switchblade.xyz/rFyRF3MMhc"></a>
+</p>
 
-*Domains used for illegal purposes will be removed and permanently banned. Please, provide a clear description of your resource in the pull request.*
+> [!WARNING]
+> *Domains used for illegal purposes will be removed and permanently banned. Please, provide a clear description of your resource in the pull request.*
```

Comment:
```suggestion
> [!WARNING]
> *Domains used for illegal purposes will be removed and permanently banned. Please provide a clear description of your resource in the pull request.*
```
1. Shouldn't be a comma there
2. The kind of people who would want to register illegal domains wouldn't care about this sort of notice anyway gud grief

Reply:
1. ill fix it
2. its better than nothing, i added this here myself